client-app/src/styles/default.scss

```diff
@@ -1,2 +1,3 @@
 @import 'bootstrap/scss/bootstrap';
-// import you custom styles here
\ No newline at end of file
+@import 'bootstrap-vue/src/index.scss';
```

Comment:
Don't import full bootstra-vue, if you need only table. Import only specific scss (you may find it in `node_modules/bootstrap-vue`)